src/main/java/by/itechart/api/entity/User.java

```diff
@@ -40,4 +40,18 @@
     @OneToOne(mappedBy = "user")
     private UserRole userRole;
 
+    @Override
```

Comment:
JFYI: You can also use `@ToString.Exclude` [annotation](https://projectlombok.org/features/ToString) to exclude some fields from string